cli/cmd/resources.go

```diff
@@ -120,6 +124,13 @@ var (
 		resourcemanager.WithApplyPreProcessor(transactionPreprocessor.Preprocess),
 	)
 
+	// deprecated resources
+	deprecatedEnvironmentClient = resourcemanager.NewClient(
+		httpClient, cliLogger,
+		"environment", "environments",
+		resourcemanager.WithProxyResource("variableset"),
```

Comment:
Used to create an entry in the registry that will point to the variable set client